vernac/metasyntax.ml

```diff
@@ -99,100 +99,103 @@ let pr_grammar = function
 
 let parse_format ((loc, str) : lstring) =
   let len = String.length str in
-  let push_token a = function
-    | cur::l -> (a::cur)::l
-    | [] -> [[a]] in
-  let push_white n l =
-    if Int.equal n 0 then l else push_token (UnpTerminal (String.make n ' ')) l in
-  let close_box i b = function
-    | a::(_::_ as l) -> push_token (UnpBox (b,a)) l
-    | _ -> user_err Pp.(str "Non terminated box in format.") in
-  let close_quotation i =
-    if i < len && str.[i] == '\'' && (Int.equal (i+1) len || str.[i+1] == ' ')
-    then i+1
-    else user_err Pp.(str "Incorrectly terminated quoted expression.") in
+  (* TODO: update the line of the location when the string contain line returns *)
```

Comment:
Maybe "newlines" instead of "line returns" ?

Reply:
OK, changed.